superset/tasks/cache.py

```diff
@@ -212,14 +212,38 @@ def get_payloads(self) -> list[dict[str, int]]:
 strategies = [DummyStrategy, TopNDashboardsStrategy, DashboardTagsStrategy]
 
 
+def fetch_csrf_token(headers: dict[str, str]) -> dict[str, str]:
```

Comment:
This isn't specific to caching - WDYT about superset.tasks.utils?

Reply:
I think moving to `superset.tasks.utils` is a splendid idea 👍